.gitlab-ci.yml

```diff
@@ -1,4 +1,4 @@
-image: quay.io/vgteam/vg_ci_prebake:latest
+image: quay.io/jonathanxu18/graviton2-tests:latest
```

Comment:
This controls the image that CI tests are run on, on our lab's shared Kubernetes cluster. Those Gitlab runners are all x86_64 and won't be able to run on an ARM image, so this would have to be at least a multi-architecture image with x86_64 in it.

Also, the Gitlab tests aren't meant to run on a Toil appliance image; they want to run on an Ubuntu-derived image with a few packages pre-installed, so we don't need to re-install the packages every time we run tests, but they don't want (and might now be able to work with) a pre-installed copy of Toil, because they need to install the Toil version we are trying to test in any given CI run.

To actually test your ARM image you probably need to add a test to the Toil tests with a `toil launch-cluster` call that deploys a cluster with ARM nodes. Possibly just as a little shell script in a new test in this file. If you haven't gotten the automatic build working on CI for the ARM version yet, you would make sure to set a `TOIL_APPLIANCE_SELF` there to the pre-built image you want to test, until you can fix it to use the current version of Toil being tested.